quipucords/tests/api/scanjob/test_scanjob.py

```diff
@@ -1997,3 +1984,46 @@ def test_filter_by_status(self, django_client, mocker):
         result = response.json()["results"][0]
         assert result["id"] == scanjob.id
         assert result["status"] == chosen_status
+
+
+@pytest.fixture
+def scanjob_with_inspect_results():
+    """ScanJob with 3 InspectResults."""
+    inspection_results = InspectResultFactory.create_batch(3)
+    scanjob = ScanJobFactory()
+    task = ScanTaskFactory(job=scanjob)
+    task.inspect_results.add(*inspection_results)
+    return scanjob
+
+
+@pytest.mark.django_db
+def test_delete_inspect_results(scanjob_with_inspect_results):
+    """Test ScanJob.delete_inspect_results method."""
+    scanjob = scanjob_with_inspect_results
+    assert InspectResult.objects.filter(tasks__job=scanjob).exists()
+    scanjob.delete_inspect_results()
+    assert not InspectResult.objects.filter(tasks__job=scanjob).exists()
+
+
+@pytest.mark.django_db
+def test_delete_inspect_results_binded_to_other_scanjobs(
```

Comment:
```suggestion
def test_delete_inspect_results_bound_to_other_scanjobs(
```

Reply:
Sorry, github auto resolved this when I accepted the suggestion.